### DISCUSSION TITLE
Incompatible with Edge Webview2

### DISCUSSION THREAD

**Discussion by pzh200707** [kimmknight/remoteapptool#107] [Q&A]
Citrix workspace application is unable to initialize because with a popup box reporting error with Edge Webview2. 

Edge is unable to be started, with no windows or error showing up

currently using: win 10, uptodate Edge

Thanks,

**Comment by way-zer** [comment]
Do you have opened Edge from local and then try to open Edge from Remote App?
May be have something to do with https://github.com/kimmknight/remoteapptool/issues/99

EDIT: Program like Edge will use named Pipe for IPC, that is shared between session. Like VSCode
![image](https://github.com/kimmknight/remoteapptool/assets/15688938/594d6867-5f8f-4995-9bbe-e3dc4ea3675f)

EDIT2: Also there are file locks, that is also shared.

**Comment by MrBrianGale** [comment]
Also, when you say you are using Win 10 and up to date Edge, is that on the server hosting the remoteapp OR the client? RemoteApps, as the name suggest, are REMOTE applications. They are RUNNING on the remote application but sending the screen updates to the client. So if the HOST doesn't have the most recent edge (for example), it could be the problem.
I am also a little confused why you'd want something like Citrix Workspace running as a remote app... That feels like you are putting layers on top of layers for accessing applications which is a bit of overkill isn't it?

I am also not sure if Citrix Workspace or WebView2 supports running under an RDP session. I have a feeling that the tool doesn't support running in that environment hence the error, but not certain. NOT all apps support running under an RDP session.
